### ISSUE TITLE
Input file needed for testing

### ISSUE THREAD

**Issue by zonca** [jluastro/maosi#2]
is it possible to make `PSFs_Imaka88-offner-10x10field_5GS8rad-Nyquist.fits` available so I can run `test.py`? if it is not too big you can directly add it to repo.

**Comment by jluastro** [comment]
The file is quite large... this is an issue we have been having with a number of repos that require large images for testing. Git LFS seems too clunky. Any suggestions/advice would be appreciated.

**Comment by zonca** [comment]
yeah, had same experience with Git LFS, if it is less than 100MB each I recommend to create another repo in your account just dedicated to those test data, and then `wget` the large file only when we need it for testing.

If more than 100MB then either host on webserver on own institution or Figshare, nice thing about Figshare is that also gives DOI and versioning
